README.Docker.md

```diff
@@ -0,0 +1,22 @@
+### Building and running your application
```

Comment:
> Unsure if this README is needed if the steps are already mentioned in the main README.
> 
> @gibbardsteve, what do you think?

Yes, this can probably be removed. It was auto-generated with docker init.